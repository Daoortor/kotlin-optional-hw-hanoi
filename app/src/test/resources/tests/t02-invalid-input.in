xsa
3a
3 1
-10000000000000000000000
1
2
15
0  1
1 4
1   2
1
1 2 3
1 3
2 3
