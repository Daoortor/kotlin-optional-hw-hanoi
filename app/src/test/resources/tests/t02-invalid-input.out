Enter tower size: Tower size should be an integer >= 2. Enter tower size: Tower size should be an integer >= 2. Enter tower size: Tower size should be an integer >= 2. Enter tower size: Tower size should be an integer >= 2. Enter tower size: Tower size should be an integer >= 2. Enter tower size: 
 █   |   | 
███  |   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: Input should contain two numbers: the indices of the source and destination stack.

 █   |   | 
███  |   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: Source stack #0 invalid

 █   |   | 
███  |   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: Destination stack #4 invalid

 █   |   | 
███  |   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: 
 |   |   | 
███  █   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: Input should contain two numbers: the indices of the source and destination stack.

 |   |   | 
███  █   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: Input should contain two numbers: the indices of the source and destination stack.

 |   |   | 
███  █   | 
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: 
 |   |   | 
 |   █  ███
___ ___ ___
[1] [2] [3]

Enter source and destination stacks: 
 |   |   █ 
 |   |  ███
___ ___ ___
[1] [2] [3]

You finished in 3 moves.
